### PR TITLE
Move more inline rolls to the new damage format

### PR DESCRIPTION
**Pull request by TikaelSol**

### CODE REVIEW

File: packs/abomination-vaults-bestiary/bloodsiphon.json [hunk 1]
```diff
@@ -232,7 +232,7 @@
...
+                    "value": "<p><strong>Requirements</strong> The bloodsiphon has a creature @UUID[Compendium.pf2e.conditionitems.Item.Grabbed] or @UUID[Compendium.pf2e.conditionitems.Item.Restrained]</p>\n<hr />\n<p><strong>Effect</strong> The bloodsiphon drains blood from the creature it has grabbed. This deals [[/r {4d4}]]{4d4 damage} and the bloodsiphon grows temporarily moist and slimy as it regains @Damage[8[healing]]{8 Hit Points}, gaining any excess as temporary Hit Points that last for 1 minute.</p>\n<p>A creature that has its blood drained by the bloodsiphon is @UUID[Compendium.pf2e.conditionitems.Item.Drained]{Drained 1} until it receives any type of healing.</p>"
```
Comment (stwlam) [inline packs/abomination-vaults-bestiary/bloodsiphon.json:235]: looks like the untyped damage there should be converted but not the healing

File: packs/abomination-vaults-bestiary/canker-cultist.json [hunk 2]
```diff
@@ -529,7 +529,7 @@
...
+                    "value": "<p>Tendrils of darkness curl out from your fingertips and race through the air. You deal 2d4 negative damage and @Damage[1[bleed]] damage to living creatures in the line. Each living creature in the line must attempt a Fortitude save.</p>\n<hr /><strong>Critical Success</strong> The creature is unaffected.\n<p><strong>Success</strong> The creature takes half the negative damage and no persistent bleed damage.</p>\n<p><strong>Failure</strong> The creature takes full damage.</p>\n<p><strong>Critical Failure</strong> The creature takes double negative damage and double persistent bleed damage.</p>\n<hr /><strong>Heightened (+1)</strong> The negative damage increases by 2d4, and the persistent bleed damage increases by 1."
```
Comment (stwlam) [inline packs/abomination-vaults-bestiary/canker-cultist.json:532]:
```suggestion
                    "value": "<p>Tendrils of darkness curl out from your fingertips and race through the air. You deal @Damage[2d4[negative],1[bleed]]{2d4 negative damage and 1 bleed damage} to living creatures in the line. Each living creature in the line must attempt a Fortitude save.</p>\n<hr /><strong>Critical Success</strong> The creature is unaffected.\n<p><strong>Success</strong> The creature takes half the negative damage and no persistent bleed damage.</p>\n<p><strong>Failure</strong> The creature takes full damage.</p>\n<p><strong>Critical Failure</strong> The creature takes double negative damage and double persistent bleed damage.</p>\n<hr /><strong>Heightened (+1)</strong> The negative damage increases by 2d4, and the persistent bleed damage increases by 1."
```

Comment (stwlam) [reply packs/abomination-vaults-bestiary/canker-cultist.json:532]: Oh, wait, this a spell? We should just remove it entirely.

Comment (TikaelSol) [reply packs/abomination-vaults-bestiary/canker-cultist.json:532]: We may want to refresh NPC copies to make sure they have working versions, but yeah.

File: packs/abomination-vaults-bestiary/bloodsiphon.json (hunk 1 above)
Comment (TikaelSol) [reply packs/abomination-vaults-bestiary/bloodsiphon.json:235]: We've been entering healing like this for a while so it was easier to find. I can undo all the healing changes to the old format.

File: packs/abomination-vaults-bestiary/canker-cultist.json (hunk 2 above)
Comment (TikaelSol) [reply packs/abomination-vaults-bestiary/canker-cultist.json:532]: Removing inlines from spells is going to be a much larger project, but one that is worth spending time on later.

**Pull request merged**